.eslintrc.js

```diff
@@ -16,11 +21,12 @@ module.exports = {
     // ts files
     {
       files: ['**/*.{gts,ts}'],
-      extends: [
-        'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
This is already part of the `@typescript-eslint/recommended` config, and we don't really need it in either case, since we include the rules ourselves now.